hazelcast-integration/openshift/client-apps/ocp-demo-frontend/pom.xml

```diff
@@ -25,7 +25,7 @@
         <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast-kubernetes</artifactId>
-            <version>1.0.0</version>
+            <version>1.5.2</version>
```

Comment:
I don't think we have a stable release for hazelcast-k8s ready for 4.0 (https://github.com/hazelcast/hazelcast-kubernetes/releases) so downgraded sample to 3.12.6.

Couldn't verify it though. Same for all other openshift samples.